core/src/main/java/io/th0rgal/oraxen/api/events/custom_block/noteblock/OraxenNoteBlockDropLootEvent.java

```diff
@@ -0,0 +1,26 @@
+package io.th0rgal.oraxen.api.events.custom_block.noteblock;
+
+import io.th0rgal.oraxen.api.events.custom_block.OraxenCustomBlockDropLootEvent;
+import io.th0rgal.oraxen.mechanics.provided.gameplay.custom_block.CustomBlockMechanic;
+import io.th0rgal.oraxen.mechanics.provided.gameplay.custom_block.noteblock.NoteBlockMechanic;
+import io.th0rgal.oraxen.utils.drops.DroppedLoot;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+public class OraxenNoteBlockDropLootEvent extends OraxenCustomBlockDropLootEvent {
+    public OraxenNoteBlockDropLootEvent(@NotNull CustomBlockMechanic mechanic, @NotNull Block block, Player player, @NotNull List<DroppedLoot> loots) {
```

Comment:
should take a noteblock mechanic directly

Reply:
Changed. On that note, OraxenNoteBlockBreakEvent also takes in CustomBlockMechanic.

Reply:
Feel free to patch that aswell, probably an oversight from copy/pasting the events originally